<!-- joy controller launch -->
<launch >
    <arg name="joy_topic_name" defalut="/joy" />
    <arg name="joy_controller_topic_name" default="/cmd_vel_joy" />
    <arg name="publish_twist" default="false" />
    <arg name="cmd_publish_hz" default="10.0" />
    <arg name="joy_velocities_coef" default="0.1" />

    <node name="joy_controller" pkg="joy_controller" type="joy_controller" >
        <param name="joy_topic_name" value="$(arg joy_topic_name)" type="string" />
        <param name="joy_controller_topic_name" value="$(arg joy_controller_topic_name)" type="string" />
        <param name="publish_twist" value="$(arg publish_twist)" type="bool" />
        <param name="cmd_publish_hz" value="$(arg cmd_publish_hz)" type="double" />
        <param name="joy_velocities_coef" value="$(arg joy_velocities_coef)" type="double" />
    </node >
</launch >
